packages/blade/src/components/Card/_decisions/decisions.md

```diff
@@ -0,0 +1,512 @@
+
+**Footer Leading Section**
+
+- `Title` & `Subtitle`
+  - Size & text variant cannot be changed
+
+**Footer Trailing Sections**
+
+- Only two action buttons can be present `primary` or `secondary`
+- Users can totally omit action buttons
+- They can also use either variants of buttons `primary` or `secondary`
+
+## API
+
+With the above constraints we can propose the Card API as follows: 
+
+Sample usage:
+
+```jsx
+<Card backgroundLevel={1}>
+  <Card.Header>
+    <Card.HeaderLeading  
+      title="Payments Links" 
+      subtitle="Share payment link via an email, SMS, messenger, chatbot etc." 
+      prefix={<CardHeaderIcon icon={DollarIcon} />}
+      suffix={<CardHeaderCounter amount={20} />}
+    />
+    <Card.HeaderTrailing 
+      visual={<CardHeaderBadge variant="positive">NEW</CardHeaderBadge>} 
+    />
+  </Card.Header>
+  <Card.Body>
+    Card Body Content
+  </Card.Body>
+  <Card.Footer>
+    <Card.FooterLeading 
+      title="Card Footer Title" 
+      subtitle="Card footer subtitle" 
+    />
+    <Card.FooterTrailing 
+      actions={{
+        primaryAction: { text: 'Know more', onClick: () => {} },
+        secondaryAction: { text: 'Read Docs', onClick: () => {} },
+      }}
+    />
+  </Card.Footer>
+</Card>
+```
+
+
+### `Card` API
+
+
+| Prop              | Type   | Default | Description                                                                            | Required |
+| ----------------- | ------ | ------- | -------------------------------------------------------------------------------------- | -------- |
+| `surfaceLevel` | `2, 3` | `2`     | Surface level of the card background color, use this based on where the card is placed |          |
+
+
+### `Card.HeaderLeading` API
+
+
+| Prop             | Type              | Default     | Description                                              | Required |
+| ---------------- | ----------------- | ----------- | -------------------------------------------------------- | -------- |
+| `title`          | `string`          | `undefined` | Title of the Card                                        | ✅        |
+| `subtitle`       | `string`          | `undefined` | Subtitle of the Card                                     |          |
+| `prefix`    | `React.ReactNode` | `undefined` | Prefix element placed before title text (restricted to: Icon)       |          |
+| `suffix`    | `React.ReactNode` | `undefined` | Suffix element placed after title text (restricted to: Counter)     |          |
+
+### `Card.HeaderTrailing` API
+
+| Prop             | Type              | Default     | Description                                              | Required |
+| ---------------- | ----------------- | ----------- | -------------------------------------------------------- | -------- |
+| `visual` | `React.ReactNode` | `undefined` | Trailing visual element placed on right side of the card (restricted to: Badge, Text, Link, IconButton) |          |
```

Comment:
`visual` seems bit odd terminology here (prefix, suffix have been used earlier).

Reply:
True that, how about using `children` here?